src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -167,6 +170,7 @@ private DataViewType GetOutputType(DataViewSchema inputSchema, HashingEstimator.
               base(Contracts.CheckRef(env, nameof(env)).Register(RegistrationName), GetColumnPairs(columns))
         {
             _columns = columns.ToArray();
+            _version = 0x00010003;
```

Comment:
>0x00010003 [](start = 23, length = 10)

I was going to suggest having a static variable containing the current version, or calling `GetVersionInfo()` here, but now that I think about it, this field can actually be of bool type, `_isVersion2` or something similar. That way it is `false` by default, and only needs to be initialized in the ctor with `ModelLoadContext`.